### ISSUE TITLE
Billing Page Re-loading the same page as the current page

### ISSUE THREAD

**Issue by Bahati308** [I-TECH-UW/OpenELIS-Global-2#1357]
# Billing Page Re-loading the same page as the current page.

1. The Page re-loads the same page as the current page. Whenever you visit any page and try to go back to the main menu and select the billing page, it loads the same page but on the new page

2. Attached  video recordings.

[Screencast from 2024-12-23 22-32-39.webm](https://github.com/user-attachments/assets/f286d5b1-a2e8-4b20-a567-4d3474597358)

3. Severity of the issue:
   1. This issue is something we would like to see improved in the future but
      does not significantly affect our work.
   2. This issue makes OpenELIS slow, difficult, or confusing, but we are able
      to keep working with it.

**Comment by Nirvanjha2004** [comment]
can i work on this ?? @Bahati308

**Comment by Bahati308** [comment]
sure, go ahead @Nirvanjha2004

**Comment by Isabirye1515** [comment]
The biling page is supposed to access the odoo integration as shown in the openELIS demo Mr
@mozzy11 , some reflection on this sir

**Comment by adityadeshlahre** [comment]
Yes @Isabirye1515 you are correct this billing page will show the all the billing service providers integration with respect to OpenELIS

**Comment by mozzy11** [comment]
In therory , that should pint to a Billing management system like ODOO. 
so we can postponde this issue until we have  the ODOO integration worked on
